content/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.rst

```diff
@@ -1,84 +1,177 @@
+13-digit barcodes to identify products.
+
+|UPC| codes can be converted to |EAN| by prefixing them with a zero. In Odoo, by default,
+|UPC|/|EAN| barcodes are automatically converted one way or another when trying to match a rule with
+the other encoding.
+
+Refer to the :ref:`default nomenclature list <barcode/operations/default-nomenclature-list>` to see
+a comprehensive list of all barcode patterns and rules to follow.
+
+Create rules
+------------
+
+|UPC| and |EAN| barcodes contain information in the barcode. Scanning these codes from the
+:ref:`default nomenclature list <barcode/operations/default-nomenclature-list>` automatically fills
+corresponding data in the Odoo database.
+
+Adding new barcode rules to this list ensures that non-standard (:dfn:`user-created`) formats are
+interpreted accurately.
+
+To create new rules, first activate :ref:`developer mode <developer-mode>`. Then, navigate to
+:menuselection:`Inventory app --> Configuration --> Barcode Nomenclatures`. Select the
+:guilabel:`Default Nomenclature` list item.
+
+On the :guilabel:`Default Nomenclature` page, select :guilabel:`Add a line` at the bottom of the
+table, which opens a window to create a new rule.
+
+.. image:: barcode_nomenclature/barcode-nomenclature-new-rule-popup.png
+   :align: center
+   :alt: Create Rules pop-up window on Default Nomenclature page.
+
+The :guilabel:`Rule Name` field is used internally to identify what the barcode represents. The
+barcode :guilabel:`Types` are different classifications of information that can be understood by the
+system (e.g., Package, Lot, Location, Coupon, etc.). The :guilabel:`Sequence` represents the
+priority of the rule; this means the smaller the value, the higher the rule appears on the table.
+
+Odoo follows the sequential order of this table, and will use the first rule it matches based on the
+sequence. The :guilabel:`Barcode Pattern` is how the sequence of letters or numbers is recognized by
+the system to contain information about the product.
 
 .. note::
-        When the barcode pattern contains .*, it means that it can contain any
-        number of characters, those characters being any number.
+   Barcode patterns can also define how numerical values, such as weight or price, are encoded in
+   the barcode.
+
+   They are indicated by **{NNN}**, where N defines where the number's digits are encoded. *Floats*
+   are also supported with the decimals, indicated by D, such as **{NNNDD}**.
+
+   In this case, the barcode field on the associated records **must** show these digits as zeroes.
+
+After filling in the information, click the :guilabel:`Save & New` button to make another rule, or
+click :guilabel:`Save & Close` to save and return to the table of rules.
+
+.. tip::
+   When the :guilabel:`Barcode Pattern` field contains `.*`, it means that it can contain **any**
+   number of characters, those characters being **any** number.
```

Comment:
```suggestion
   When the :guilabel:`Barcode Pattern` field contains `.*`, it means it can contain **any**
   number of characters - those characters being **any** number.
```